### DISCUSSION TITLE
"Page not found" when primary key's "Display" is set to "related value"

### DISCUSSION THREAD

**Discussion by powerseed** [directus/directus#10357] [Archive]
I have a collection where its PK is also a FK.

When I set its "Display" to "Related Value" as shown below, an "Page not found" error always shows up after clicking a row of this table. (The table's collection view works fine though.)

![image](https://user-images.githubusercontent.com/42278237/144961115-507e480d-2b2c-4c50-93d9-504d7cf5eeae.png)

And there is no error being displayed in the console.

If I set its "Display" to "Raw Value", then the error disappears and everything works fine.

Not sure if this is a bug. Please help.

Thanks in advance!

**Comment by rijkvanzanten** [comment]
> Not sure if this is a bug.

Good question! I've never tried such a setup. What does your schema look like? Could you share a basic ERD or schema snapshot from Directus?
